node/src/Commands.ts

```diff
@@ -4,13 +4,13 @@
 
 import { createLeakedStringVec, MAX_REQUEST_ARGS_LEN } from "glide-rs";
 import Long from "long";
-import { LPosOptions } from "./commands/LPosOptions";
 import { FlushMode } from "./commands/FlushMode";
+import { LPosOptions } from "./commands/LPosOptions";
```

Comment:
Do you have additional linter which reorders imports?

Reply:
It might be my VS Code setting which automatically reorders imports.
Do you want me to discard these changes?

Reply:
No, I like that.
If you can identify which setting/plugin does that, please update developer guide, so everyone will use that.